src/my_project/tagging/metadata_controller.py

```
@@ -256,15 +240,17 @@ def get_embedded_metadata(self) -> ComicInfo:
             return extractor.run()
 
     def clean_embedded_metadata(self, raw_data: ComicInfo) -> ComicInfo:
-        with MetadataProcessing(raw_data) as cleaner:
+        with MetadataProcessing(raw_data, self.config_manager) as cleaner:
             try:
                 cleaned_comic_info = cleaner.run()
                 # new_name, publisher_int = cleaner.new_filename_and_folder()
                 # ! Take this function from metadata_cleaning and use in this class.
                 return cleaned_comic_info
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)